share/locale/languages.json

```diff
@@ -1,72 +1,216 @@
 {
-    "af": "Afrikaans",
-    "ar": "العربية",
-    "ar_DZ": "\u200e(العربية (الجزائر",
-    "ar_EG": "\u200e(العربية (مصر",
-    "ar_SD": "\u200e(العربية (السودان" ,
-    "ast": "Asturianu",
-    "be": "Беларуская мова",
-    "bg": "български",
-    "br": "Brezhoneg",
-    "ca": "Català",
-    "ca@valencia": "Valencià",
-    "cs": "Čeština",
-    "cy": "Cymraeg",
-    "da": "Dansk",
-    "de": "Deutsch",
-    "el": "ελληνικά",
-    "en_GB": "English (GB)",
-    "en_US": "English (US)",
-    "eo": "Esperanto",
-    "es": "Español",
-    "et": "Eesti",
-    "eu": "Euskara",
-    "fa": "فارس",
-    "fi": "Suomi",
-    "fil": "Filipino",
-    "fo": "Føroyskt",
-    "fr": "Français",
-    "ga": "Gaeilge",
-    "gd": "Gàidhlig",
-    "gl": "Galego",
-    "he": "עברית",
-    "hi_IN": "हिन्दी",
-    "hr": "Hrvatski",
-    "hu": "Magyar",
-    "hy": "Հայերեն",
-    "id": "Bahasa Indonesia",
-    "ig": "Ásụ̀sụ̀ Ị̀gbò",
-    "it": "Italiano",
-    "ja": "日本語",
-    "ka": "ქართული",
-    "kab": "ⵜⴰⵇⴱⴰⵢⵍⵉⵜ",
-    "ko": "한국어",
-    "lt": "Lietuvių",
-    "lv": "Latviešu",
-    "ml": "മലയാളം",
-    "mn_MN": "Монгол хэл",
-    "mt": "Malti",
-    "nb": "Norsk (Bokmål)",
-    "nl": "Nederlands",
-    "nn": "Norsk (Nynorsk)",
-    "pl": "Polski",
-    "pt": "Português",
-    "pt_BR": "Português brasileiro",
-    "ro": "Română",
-    "ru": "Русский",
-    "scn": "Sicilianu",
-    "sk": "Slovenčina",
-    "sl": "Slovenščina",
-    "sr": "Srpski",
-    "sr_RS" : "Српски",
-    "sv": "Svenska (modern)",
-    "sv_SE": "Svenska (traditionell)",
-    "th": "ภาษาไทย",
-    "tr": "Türkçe",
-    "uk": "Українська",
-    "uz@Latn": "O'zbek tili",
-    "vi": "Tiếng Việt",
-    "zh_CN": "简体中文",
-    "zh_HK": "繁體中文 (香港)",
-    "zh_TW": "繁體中文 (台灣)"
+    "af": {
+        "name": "Afrikaans"
+    },
+    "ar": {
+        "name": "العربية"
+    },
+    "ar_DZ": {
+        "name": "\u200e(العربية (الجزائر",
+        "fallbackLanguages": ["ar"]
+    },
+    "ar_EG": {
+        "name": "\u200e(العربية (مصر",
+        "fallbackLanguages": ["ar"]
+    },
+    "ar_SD": {
+        "name": "\u200e(العربية (السودان",
+        "fallbackLanguages": ["ar"]
+    },
+    "ast": {
+        "name": "Asturianu"
+    },
+    "be": {
+        "name": "Беларуская мова"
+    },
+    "bg": {
+        "name": "български"
+    },
+    "br": {
+        "name": "Brezhoneg"
+    },
+    "ca": {
+        "name": "Català"
+    },
+    "ca@valencia": {
+        "name": "Valencià"
+    },
+    "cs": {
+        "name": "Čeština"
+    },
+    "cy": {
+        "name": "Cymraeg"
+    },
+    "da": {
+        "name": "Dansk"
+    },
+    "de": {
+        "name": "Deutsch"
+    },
+    "el": {
+        "name": "ελληνικά"
+    },
+    "en_GB": {
+        "name": "English (GB)",
+        "fallbackLanguages": ["en_US"]
+    },
```

Comment:
I wonder is we can now add en_AU and en_NZ, both falling back to en_GB?
Maybe even en_CD?

Reply:
This would enable us to reduce the en_GB translations down to those strings where British English differes from American/International English, some 440 strings